jobs/e2e-test.groovy

```diff
@@ -616,6 +613,15 @@ onWorker(WORKER_TYPE, '5h') {     // timeout
          stage("Analyzing results") {
             analyzeResults();
```

Comment:
Ah ha, ok! So I think the critical thing here is that the results will get reported from run-tests, which will unblock the deploys, and then the cypress tests will run after, separately.

Reply:
I'm not sure if that's quite correct... It's possible i'm wrong, but I think we need to exit the `notify` block, i think it's the `buildmaster` bit of that that kicks off the message to users to finish up?

Reply:
hmm yeah that makes sense. Maybe @csilvers also have a good idea about that?